draft-ietf-webpush-protocol.xml

```diff
@@ -1137,6 +1147,12 @@ HEADERS      [stream 82] +END_STREAM
           This could force the user agent to serialize subscription requests to the push
           service.
         </t>
+        <t>
+         A user agent that monitors subscriptions for other clients might

```

Comment:
@costinm - I understand that moving a subscription between UA is tricky and I don't want to resolve it in this draft.
I just wanted to expand the third paragraph of 4.1 to make the intent clearer (there was come comments from Kit Cambridge that showed it was not clear enough).
Possibly, we could also add the multi-user case to 7.5.